tests/static/js/manual-initializer.js

```diff
@@ -0,0 +1,32 @@
+import ManualInitializer from '../../../static/js/manual-initializer';
+
+describe('The manual initializer works', () => {
+  let initAnswers;
+
+  beforeEach(() => {
+    global.window.AnswersExperience = {
```

Comment:
is the global necessary?

Reply:
based on the is-staging tests, I thought it was but it turns out it's not. Updated